internal/services/springcloud/spring_cloud_gateway_resource.go

```diff
@@ -263,6 +270,36 @@ func (s SpringCloudGatewayResource) Arguments() map[string]*pluginsdk.Schema {
 			},
 		},
 
+		"response_cache": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"cache_type": {
```

Comment:
cache is already part of the block name?
```suggestion
					"type": {
```